tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -14,30 +14,15 @@
 //
 //******************************************************************************
 
-#include <TestFramework.h>
+#import <TestFramework.h>
 #import <Foundation/Foundation.h>
-#import "NSBooleanPredicate.h"
 
 TEST(NSPredicate, Init) {
     NSPredicate* predicate = [[NSPredicate alloc] init];
     ASSERT_TRUE_MSG(predicate != nil, "FAILED: predicate should be non-null!");
     [predicate release];
 }
 
-TEST(NSPredicate, NSBooleanPredicateYES) {
```

Comment:
NSBooleanPredicate is our own concoction, and does not exist on OSX

---
In reply to: [72315140](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72315140) [](ancestors = 72315140)